aspnetcore/release-notes/aspnetcore-9/samples/WebSocketsKeepAliveTimeoutExample/Program.cs

```diff
@@ -0,0 +1,46 @@
+#define KEEP_ALIVE_GLOBAL   //KEEP_ALIVE_PER_WEBSOCKET
+
+#if KEEP_ALIVE_GLOBAL
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddControllers();
+
+var app = builder.Build();
+
+// <snippet_WebSocket_KeepAliveInterval_Global>
+app.UseWebSockets(new WebSocketOptions { KeepAliveInterval = TimeSpan.FromSeconds(15) });
```

Comment:
Whoops, I had a typo in my code example, this should be `KeepAliveTimeout`

Reply:
Thanks.  I thought the issue was that I didn't have the very latest bits.  I will change, test and update

Reply:
Oh no wait, it was `KeepAliveInterval` that would compile but `KeepAliveTimeout` that would not. So I guess I didn't have the latest bits.  Will change global example to `KeepAliveTimeout`.